package.json

```diff
@@ -40,6 +40,7 @@
     "eslint-plugin-import": "^2.26.0",
     "nano-staged": "^0.5.0",
     "p-limit": "^4.0.0",
+    "pkg-pr-new": "^0.0.2",
```

Comment:
let's remove this and just try the pnpx pkg-pr-new publish command

this way, pnpx will bring it automatically